### PR TITLE
Created a verification prompt screen in the component page

### PR DESCRIPTION
**Pull request by Zer0SZN**

### CODE REVIEW

File: src/components/VerificationOverlay.js
Comment (annonymousauthority) [inline src/components/VerificationOverlay.js:1]: This component should collect an email prop that's what's needed to display to the user.

File: src/components/VerificationOverlay.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const VerificationOverlay = () => {
+  return (
+    <div className="absolute p-2">
+      <div className="space-y-6 w-full bg-neutral-100 p-9 rounded-2xl ">
+        <div className="text-blue-600 dark:text-white text-3xl font-extrabold">
+          Account Verification
+        </div>
+        <p>
+          A verification link has been sent to{" "}
+          <span className="text-blue-600">augustine@example.com.</span> Please
```
Comment (annonymousauthority) [inline src/components/VerificationOverlay.js:10]: Please change this to be the prop `{email}`

Comment (annonymousauthority) [reply src/components/VerificationOverlay.js:10]: Also remove `please check your email` to another line

File: src/components/VerificationOverlay.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          check your inbox and click on the link to complete your registration.
+        </p>
+        <div>
```
Comment (annonymousauthority) [inline src/components/VerificationOverlay.js:13]: Remove from line 13 to line 17

File: src/components/VerificationOverlay.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          <a href="#" className="text-blue-600 underline  font-bold">
+            Change Email
+          </a>
+        </div>
+        <div className="flex flex-col space-y-3">
+          <button className="bg-blue-600 hover:bg-blue-600 text-white px-6 py-3 rounded-md">
```
Comment (annonymousauthority) [inline src/components/VerificationOverlay.js:19]: This should be an anchor not a button

Comment (annonymousauthority) [reply src/components/VerificationOverlay.js:19]: The anchor `<a href="">` should take user to their default email app or browser mail

File: src/components/VerificationOverlay.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            Open mail app
+          </button>
+        </div>
+        <p className="text-neutral-500">
+          Did not receive the mail? Check your spam
+          <span className="absolute  right-12">
```
Comment (annonymousauthority) [inline src/components/VerificationOverlay.js:25]: Remove line 25 to 27

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

File: src/components/VerificationOverlay.js (hunk 1 above)
Comment (Zer0SZN) [reply src/components/VerificationOverlay.js:10]: Oh Ok I was actually follow accordingly to the design

File: src/components/VerificationOverlay.js (hunk 4 above)
Comment (Zer0SZN) [reply src/components/VerificationOverlay.js:25]: That will be removing the resend mail timer

Comment (Remi-dee) [reply src/components/VerificationOverlay.js:25]: Hi @Zer0SZN please if you’re still having issues implementing these changes, kindly reach me.

File: src/components/VerificationOverlay.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+"use client";
+
+const VerificationOverlay = ({ email }) => {
+  //testing callback for email
+  email = "kharunaking@gmail.com";
```
Comment (annonymousauthority) [inline src/components/VerificationOverlay.js:5]: Remove this along with all comments

File: src/components/VerificationOverlay.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+        </p>
+        <p>
+          Please check your inbox and click on the link to complete your
+          registration.
+        </p>
+        <div className="flex flex-col space-y-3">
+          <a
+            className="bg-blue-600 hover:bg-blue-500 text-white px-6 py-3 rounded-md text-center"
+            href="#"
+            onClick={redirectToEmail}
```
Comment (annonymousauthority) [inline src/components/VerificationOverlay.js:30]: Remove this. `onClick` is useless in this code

File: src/components/VerificationOverlay.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+          <span className="text-blue-600">{email}.</span>
+        </p>
+        <p>
+          Please check your inbox and click on the link to complete your
+          registration.
+        </p>
+        <div className="flex flex-col space-y-3">
+          <a
+            className="bg-blue-600 hover:bg-blue-500 text-white px-6 py-3 rounded-md text-center"
+            href="#"
```
Comment (annonymousauthority) [inline src/components/VerificationOverlay.js:29]: you href should look like this `href={`mailto:${email}`}`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
That's everything

**Comment by Zer0SZN** [comment]
Review pr

**Review by annonymousauthority — APPROVED** [review]
LG

**Comment by Zer0SZN** [comment]
F7615456-16CA-4CFB-ABD5-97ADF5AF66DA_1_105_c

**Pull request merged**